Libs/GObject/Classes/Object.cs

```diff
@@ -128,12 +129,12 @@ private void Initialize(IntPtr ptr)
             RegisterProperties();
             RegisterOnFinalized();
 
-            // Allow subclasses to perform initialization
+            // Allow subclasses to perform initialisation
```

Comment:
Typo regression

Reply:
While this is correct British English, I'm not that petty 😅.

I imagine this is another regression from the merge. I'll push a commit that reverts these.